docs/docs/behavior/hold-tap.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Hold-tap behavior
+sidebar_label: Hold-Tap
+---
+
+## Summary
+Hold-tap is the basis for other behaviors such as layer-tap and mod-tap. 
+
+Simply put, the hold-tap key will output the 'hold' behavior if it's held for a while, and output the 'tap' behavior when it's tapped quickly.
+
+
+### Hold-Tap
+The `tapping_term_ms` parameter decides between a 'tap' and a 'hold'.
+
+![Simple behavior](../assets/hold-tap/case1_2.png)
+
+By default, the hold-tap is configured to also select the 'hold' functionality if another key is tapped while it's active: 
+
+![Hold preferred behavior](../assets/hold-tap/case1_2.png)
+
+We call this the 'hold-preferred' flavor of hold-taps. While this flavor may work very well for a ctrl/escape key, it's not very well suited for home-row mods or layer-taps. That's why there are two more flavors to choose from: 'tap-preferred' and 'balanced'.
+
+![Hold-tap comparison](../assets/hold-tap/comparison.png)
+
+### Configuration
+A code example which configures a mod-tap setting that works with homerow mods:
+
+```
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+
+/ {
+	behaviors {
+		&hm: homerow_mods {
```

Comment:
Extra `&` here.

Also, since we already *register* this elsewhere by way of the `behaviors.dtsi`, I think it may be more confusing than helpful to have this added like this in the docs?

Reply:
I've removed the "homerow mods" default binding instead of removing it from the docs, I think the homerow mods can be done in too many ways to have it make sense to make a default behavior.